content/03-tea/05-breakdown-view.md

```diff
@@ -3,16 +3,16 @@ title       = "View"
 weight      = 5
 +++
 
-The next part is the view:
+The next part is the `view` function:
 
 ```elm
 view : Model -> Html Msg
 view model =
     div [] [ text model ]
 ```
 
-This is a function that take a `Model` and return `Html`.
+This is a function that takes a `Model` argument and returns `Html`.
 
-Note `Html Msg`. `Msg` is a type variable. This means that this `Html` emits messages of a concrete type `Msg`. 
+If you’re wondering about the `Html Msg`. The `Msg` here is called a type variable. In this example, it indicates that the `Html` that our `view` returns  will only ever emit the messages specified in our program’s `Msg` type.  Being specific about this enables the Elm compiler to guarantee that our program handles all possible messages.
```

Comment:
Hmm, trying to explain the difference between `Html msg` and `Html Msg` seems overly confusing for a complete beginner audience. You'll note I removed all the generic types I could from the example, specifically to explaining this issue.

I'll see if I can write something about the `Html` type taking another type as an argument, and only mention generics to say “If you're wondering if this is the generic types you’re familiar with from Rust or Java, you’re right.”